samples/amber.cc

```diff
@@ -203,6 +207,13 @@ std::string ReadFile(const std::string& input_file) {
   return std::string(data.begin(), data.end());
 }
 
+class Delegate : public amber::Delegate {
+ public:
+  void log(std::string command) {
+    std::cout << "Graphics API call: " << command << std::endl;
```

Comment:
Do we need "Graphics API call: " in here or can we just log the command?

Reply:
This was to convince myself that the log I saw was from the delegate function, let's just print the command indeed.